docs/RELEASE.md

```diff
@@ -0,0 +1,66 @@
+# Managing Releases
+
+
+## Handling patch releases
+
+### When should I create a patch release?
+Patch releases are exclusively for bugs and and security fixes. Patch releases can have minor _enhancements_ to 
+existing features but really should not affect the API in any noticeable way (unless it is adding an argument). For any
+functionality changes or new endpoints, please create a minor or major release.
+
+A patch release should increment the third digit e.g. `0.6.9 -> 0.6.10`. This indicates to users that the release 
```

Comment:
@dimberman, `This indicates to users that the release` is this complete statement?